.github/workflows/bump-npm-version.yml

```diff
@@ -0,0 +1,60 @@
+name: Bump NPM version (core/angular/react)
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release type'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+
+jobs:
+  release-angular:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fail if branch is not main
+        if: github.event_name == 'workflow_dispatch' && github.ref != 'refs/heads/main'
```

Comment:
I believe it should be 'develop' as we're branching from develop when preparing a release, and then merging in to main when we are ready to release

Reply:
Correcto!